client/client.go

```diff
@@ -293,6 +322,22 @@ func (c *simpleHTTPClient) Do(ctx context.Context, act httpAction) (*http.Respon
 	return resp, body, err
 }
 
+type authorizingHTTPClient struct {
```

Comment:
i cannot find where we used this. i think we are using the authedAction instead?